### DISCUSSION TITLE
I need my project to not use service worker, how can I do that?

### DISCUSSION THREAD

**Discussion by LeslieZhang** [vite-pwa/vite-plugin-pwa#735] [Q&A] [answered]
My current project is configured like this：
`
VitePWA({
        registerType: 'autoUpdate',
        manifest: {
          name: 'demo',
          id: 'demo',
          short_name: 'demo',
          description: 'demo',
          // theme_color: '#DC143C',
          // background_color: '#FFFF00', 
          display: 'fullscreen',
          icons: [
            {
              src: 'pwaLogo.png',
              sizes: '144x144', // 大于144
              type: 'image/png'
            }
          ]
        },
        devOptions: {
          enabled: true
        }
      })
`
After publishing to the production environment, the browser's js file is not updated after refreshing the page multiple times.
Found the following picture：
![image](https://github.com/user-attachments/assets/3b453a01-bc13-42c5-b549-cc31d42f23a6)
**From service worker**

I want to use vite-plugin-pwa when my project does not have any cache and does not exist From service worker。

I would really appreciate if someone could answer my question, thank you very much

**Comment by userquin** [comment, marked as answer]
Review the cache headers for the entry point and the service worker: https://vite-pwa-org.netlify.app/deployment/

If you want to remove the service worker check https://vite-pwa-org.netlify.app/guide/unregister-service-worker.html

**Reply by userquin** [reply]
If you want to exclude some urls from sw interception, you can use `workbox.navigateFallbackDenylist` regexp array.

When offline, the browser will show the default offline page when navigating to any page included in previous regexp array: check https://github.com/vite-pwa/vite-plugin-pwa/discussions/734 and https://github.com/vite-pwa/sveltekit/issues/65.
